services/fabricv4/client.go

```diff
@@ -683,20 +683,32 @@ func (e GenericOpenAPIError) Model() interface{} {
 	return e.model
 }
 
-// format error message using title and detail when model implements rfc7807
+// format error message using title and detail when model is an instance of
+// the Error component schema or when it implements rfc7807
 func formatErrorMessage(status string, v interface{}) string {
 	str := ""
-	metaValue := reflect.ValueOf(v).Elem()
+	errorModel, ok := v.(*Error)
```

Comment:
Looks like this openapi-generator doesn't generate a dedicated `ErrorList` model, so `v` is `[]Error`: https://github.com/equinix/equinix-sdk-go/blob/main/services/fabricv4/api_connections.go#L123

Might want to check for both just to be safe:  try casting to `[]Error`, if that fails try cast to `Error`, if that still fails fall through to the pre-existing logic.